### PR TITLE
[Platform-Auth] Complete Rework

### PR DESCRIPTION
**Pull request by npapapietro**

### CODE REVIEW

File: charts/platform-auth/templates/_helpers.tpl [hunk 1]
```diff
@@ -5,6 +5,26 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{- define "platform-auth.helper.codecentricName" -}}
```
Comment (cameojokes) [inline charts/platform-auth/templates/_helpers.tpl:8]: Make these like `platform-auth.helper.codecentric.name`

File: charts/platform-auth/templates/_keycloak.tpl [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  - "--http-enabled=true"
+  - "--http-port=8080"
+  - "--hostname-strict=false"
+  - "--hostname-strict-https=false"
+  - "--spi-events-listener-jboss-logging-success-level=info"
+  - "--spi-events-listener-jboss-logging-error-level=warn"
+dbchecker:
+  enabled: true
+database:
+#   existingSecret: "auth-pguser-keycloak"
```
Comment (cameojokes) [inline charts/platform-auth/templates/_keycloak.tpl:52]: What's this?

File: charts/platform-auth/templates/_keycloak.tpl [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  vendor: postgres
+  hostname: {{ include "platform-auth.helper.postgresName" $ }}
+  port: "5432"
+#   username: "keycloak"
```
Comment (cameojokes) [inline charts/platform-auth/templates/_keycloak.tpl:56]: And this?

File: charts/platform-auth/values.yaml [hunk 4]
```diff
@@ -2,81 +2,108 @@
...
+      valuesOverride: {}
+
+  traefik:
+    enabled: true
+    namespace: traefik
+    chart:
+      url: "https://traefik.github.io/charts"
+      name: "traefik"
+      version: "20.1.1"
+# keycloak:
```
Comment (cameojokes) [inline charts/platform-auth/values.yaml:68]: Do we need these?

**Review by cameojokes — CHANGES_REQUESTED** [review]

File: charts/platform-auth/values.yaml (hunk 4 above)
Comment (npapapietro) [reply charts/platform-auth/values.yaml:68]: yes

**Pull request merged**